### ISSUE TITLE
why i can't load a plan.xml? 

### ISSUE THREAD

**Issue by lucici1999** [matsim-org/matsim-code-examples#1022]
![image](https://github.com/matsim-org/matsim-code-examples/assets/68318084/77680b9b-2512-4689-bb39-9bc7ed883e94)
and sometimes it keeps loading like this

![image](https://github.com/matsim-org/matsim-code-examples/assets/68318084/68a64a79-b4d9-4758-b18c-e02a8ab5b38f)
This is my different plan.xml, but they can't work neither
![image](https://github.com/matsim-org/matsim-code-examples/assets/68318084/dec433a8-e570-48bf-8566-bd0e7bb4d3a3)
![image](https://github.com/matsim-org/matsim-code-examples/assets/68318084/3d5f2c7a-64d0-4e76-93a6-c32097ba165f)

**Comment by Janekdererste** [comment]
Could you please provide your plans file as an actual file?

From the screenshot it looks like your file is incomplete, but it is hard to tell because it is an image. In general it is easier for us to not have text as screenshots.

**Comment by lucici1999** [comment]
> Could you please provide your plans file as an actual file?
> 
> From the screenshot it looks like your file is incomplete, but it is hard to tell because it is an image. In general it is easier for us to not have text as screenshots.

yes,this is my file ,I  tried links and nodes ,two different ,but it still not works
this is file1:(route type="links")
**<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE plans SYSTEM "http://www.matsim.org/files/dtd/plans_v4.dtd">
<plans>
	<person id="2660000000405910" >
		<plan selected="yes">
			<act type="home" link="24446" x="13375682.69" y="4290362.065" start_time="00:00" dur="08:00" end_time="08:00" ref_id="6" />
			<leg num="0" mode="pt" dep_time="08:00" trav_time="00:30" arr_time="08:30">
				<route type="links" trav_time="00:30">
					 8741327474 8740771585 8740771595 
				</route>
			</leg>
			<act type="work" link="24438" x="1.337555134E7" y="4290693.919" start_time="08:30" dur="08:00" end_time="16:30" ref_id="917" />
			<leg num="1" mode="pt" dep_time="16:30" trav_time="00:30" arr_time="17:00">
				<route type="links" trav_time="00:06">
					8741327466 8740771595 8740771585 8740771585 3928570845
				</route>
			</leg>
			<act type="home" link="24446" x="13375682.69" y="4290362.065" start_time="17:00" dur="07:00" end_time="24:00" ref_id="6" />
		</plan>
	</person>
</plans>**
and this is file 2(route type=nodes)
_**<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE plans SYSTEM "http://www.matsim.org/files/dtd/plans_v4.dtd">

<plans>
	<person id="2660000000337440" >
		<plan selected="yes">
			<act type="home" link="16938" x="13379038.31" y="4296490.766" start_time="00:00" dur="08:00" end_time="08:00" ref_id="6" /><!濠洼>
			<leg num="0" mode="pt" dep_time="08:00" trav_time="00:03" arr_time="08:03">
				<route trav_time="00:03">
					 3265502533 3265502956 3265502956 8957791933 3265502573 3265502542 2914709922 3413682258 8258914751 3413682253 6902632501 9060498480 3413682276 3413682244 3685215371 3413682242 3685215407 3685215446 3685215457 941471903 4491195653 4491195627 2325218659 3685215418 
				</route>
			</leg>
			<act type="jiaotongcenter" link="7007" x="13383391.12" y="4296073.459" start_time="07:10" dur="00:01" end_time="07:11" ref_id="917" />
			<leg num="1" mode="pt" dep_time="07:11" trav_time="00:22" arr_time="07:33">
				<route trav_time="00:22">
					3685215418 5554157753 5554157754 3265469752 3265469700 3265469559 5431887256 4476657086 4476657087 4249903555 3265455687 3265455411 3265455405 3265455598 3265455420 3265455629 3265455472
				</route>
			</leg>
			<act type="work" link="19696" x="13388557.8" y="4298122.873" start_time="07:33" dur="16:27" end_time="24:00" ref_id="6" /><!北船重工东门>
		</plan>
	</person>

</plans>**_


thank you very much!

**Comment by lucici1999** [comment]
> Could you please provide your plans file as an actual file?
> 
> From the screenshot it looks like your file is incomplete, but it is hard to tell because it is an image. In general it is easier for us to not have text as screenshots.
file 1 picture
![image](https://github.com/matsim-org/matsim-code-examples/assets/68318084/d6a4905a-05b4-44d8-98c7-6a635489801d)
file 2 picture
![image](https://github.com/matsim-org/matsim-code-examples/assets/68318084/dd383f93-b5bf-488d-9ee9-58d23a2350b3)

**Comment by Janekdererste** [comment]
I am very sorry, but these are still screenshots. 

If you attach your file here, I will try myself. This way I cannot tell you what is wrong.

**Comment by mrieser** [comment]
As the problem is in Via, there might be an exception in the log file.

- Share your actual plan.xml-file here so we can try to reproduce it.

- Or reproduce it yourself, then, in Via, go to menu Help > Report Bug/Feedback, and click on "Open Directory" to access the logfile of Via. Send the Logfile to the stated email address or attach it here, so we can have a look at it.

**Comment by lucici1999** [comment]
> I am very sorry, but these are still screenshots.
> 
> If you attach your file here, I will try myself. This way I cannot tell you what is wrong.

I'm sorry I've been a bit busy lately and haven't checked my email. I can't upload my xml file to this, I copied and pasted the text on this reply,
![image](https://github.com/matsim-org/matsim-code-examples/assets/68318084/88f02faa-f07d-4178-99ee-03c334703e39)
but it won't show up for some reason. I'm sorry and thank you. I'm going to write a new xml and try again

**Comment by lucici1999** [comment]
> As the problem is in Via, there might be an exception in the log file.
> 
> * Share your actual plan.xml-file here so we can try to reproduce it.
> * Or reproduce it yourself, then, in Via, go to menu Help > Report Bug/Feedback, and click on "Open Directory" to access the logfile of Via. Send the Logfile to the stated email address or attach it here, so we can have a look at it.

I will try these methods，and thanks for your advice.
